tests/glm/test_glm.py

```diff
@@ -1735,6 +1747,53 @@ def test_std_errors(regression_data):
     np.testing.assert_allclose(ourse, smse, rtol=1e-8)
 
 
+@pytest.mark.parametrize("fit_intercept", [True, False])
+@pytest.mark.parametrize("family", ["poisson", "normal", "binomial"])
+def test_array_std_errors(regression_data, classification_data, family, fit_intercept):
+    X, y = regression_data
+    if family == "poisson":
+        y = np.round(abs(y))
+        sm_family = sm.families.Poisson()
+        dispersion = 1
+    elif family == "binomial":
+        X, y = classification_data
```

Comment:
Can't you just transform the outcome to be binary here (as for Poisson) instead of adding a fixture?